docs/maintainer/infrastructure.md

```diff
@@ -388,3 +418,226 @@ If this is not done, then the output validation process will block the package f
+- [`conda-forge/automerge-action`](https://github.com/conda-forge/automerge-action)
+- [`conda-forge/github-app-token`](https://github.com/conda-forge/github-app-token) (fork)
+- [`conda-forge/webservices-dispatch-action`](https://github.com/conda-forge/webservices-dispatch-action)
+
+### Continuous integration
+
+:::tip See also
+Refer to the [`conda-forge.yml` documentation](/docs/maintainer/conda_forge_yml/) to learn how to configure your CI providers.
+:::
+
+#### Azure Pipelines
+
+- 🌐 https://dev.azure.com/conda-forge/feedstock-builds/_build
+- 📍 Available on all feedstocks
+- 🛠 Provides [Microsoft-hosted runners](https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml) (x64 Linux, macOS and Windows)
+- 🔒 Needs access to Anaconda.org (cf-staging)
+
+conda-forge benefits from the generously offered Microsoft-hosted runners.
+
+#### Travis CI
+
+- 🌐 https://app.travis-ci.com/github/conda-forge
+- 📍 Available on all feedstocks
+- 🛠 Provides [native Linux aarch64 and ppc64le runners](https://docs.travis-ci.com/user/reference/overview/)
+- 🔒 Needs access to Anaconda.org (cf-staging)
+
+#### Cirun
+
+- 🌐 https://cirun.io
+- 📍 Available on selected feedstocks only
+- 🛠 Provides several architectures (depending on feedstock configuration)
+- 🔒 Needs access to Anaconda.org (cf-staging) and the configured backend
+
+Configured with `@conda-forge-daemon`.
+
+Organization-wide configuration can be found in the [`.cirun` repository](https://github.com/conda-forge/.cirun).
+
+#### Github Actions
+
+- 🌐 https://github.com/features/actions
+- 📍 Not available in feedstocks (only admin tasks)
+- 🛠 Provides [GitHub-hosted runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners) (x64 Linux, macOS and Windows)
+- 🔒 Has access to Github API
+
+#### Retired services
+
+- [AppVeyor](https://ci.appveyor.com/account/conda-forge/projects)
+- Circle CI
+- Drone.io
+
+### Delivery and distribution
+
+#### Anaconda.org
+
+- 🌐 Channels/organizations: [`cf-staging`](https://anaconda.org/cf-staging/dashboard), [`conda-forge`](https://anaconda.org/conda-forge/dashboard)
+- ⛓ Used by feedstocks
+
+#### Docker Hub
+
+- 🌐 https://hub.docker.com/u/condaforge/
+- ⛓ Used by [`conda-forge/docker-images`](#docker-images), [`conda-forge/miniforge-images`](https://github.com/conda-forge/miniforge-images)
+
+#### Github Packages
+
+- 🌐 https://github.com/orgs/channel-mirrors/packages
+- ⛓ Used by [`channel-mirrors/conda-oci-mirror`](https://github.com/channel-mirrors/conda-oci-mirror)
+
+#### Github Releases
+
+- 🌐 https://github.com/conda-forge/miniforge/releases
+- ⛓ Used by [`conda-forge/miniforge`](https://github.com/conda-forge/miniforge)
+
+#### Quay
+
+- 🌐 https://quay.io/organization/condaforge
+- ⛓ Used by [`conda-forge/docker-images`](#docker-images)
+
+### Servers
+
+#### Heroku
+
+- 🌐 https://hub.docker.com/u/condaforge/
```

Comment:
I think this URL Is wrong.

Reply:
Do you know the right one, by chance?

Reply:
To be precise: I tried https://conda-forge.herokuapp.com but only  got a 404, though that maybe because I didn't use a proper endpoint?

Reply:
That url is correct but we don't serve an endpoint there. See for example this one:

https://conda-forge.herokuapp.com/status-monitor

To be honest, that app is a service backend and not meant to be called directly by a human.

Reply:
Is there something like an admin interface that would be more meaningful to link to here? Even though that wouldn't be accessible by users, of course, I have got the impression that core members reference this page as much as any one else...

Reply:
This is the url the status page uses: https://conda-forge.herokuapp.com/alive

Reply:
The only one that displays data is the status-monitor one IIRC. So let's link that!